src/ids/update_pci_usb

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+
+# Script to update pci and usb id data.
+#
+# Just run it.
+
+[ -x convert_hd -a -d src ] || {
+  echo "this script must be run in the 'src/ids' directory"
+  exit 1
+}
+
+curl -o pci.ids http://pci-ids.ucw.cz/v2.2/pci.ids || {
+  echo "oops, no pci ids"
+  exit 1
+}
+
+curl -o usb.ids http://www.linux-usb.org/usb.ids || {
+  echo "oops, no usb ids"
+  exit 1
+}
+
+# keep only device info in usb list
+perl -pi -e 'undef $_ if /^C\s/..1' usb.ids
```

Comment:
OMG is that a flip-flop operator? PLEASE add an explaining comment or use another construct.

Reply:
thanks! I thought that if /foo/..1 would end soon because 1 is true. but then man perlop says something about line numbers... Perl at its best